app-text/sablotron/files/1.0.3-bind-with-lsupcxx.patch

```diff
@@ -0,0 +1,18 @@
+Date: Thu, 14 Sep 2023 09:59:35 +0200
+Subject: [PATCH] bind with lsupc++
+
+Signed-off-by: Pascal Jäger pascal.jaeger@leimstift.de
+
+--- a/configure.in
++++ b/configure.in
+@@ -354,7 +354,7 @@ dnl miscellaneous - initial
+ dnl vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
+
+ dnl here we collect needed libraries
+-OTHER_LIBS=
++OTHER_LIBS="-lsupc++ "
```

Comment:
@thesamesam It this patch actually necessary? 
If I bind with `CC=gcc CXX=gcc` this fails because gcc does not automatically bind with lsupc++.
If I bind with `CC=clang CXX=clang` it fails as well. 

But `CC=clang CXX=clang++` works as well as `CC=gcc CXX=g++`

Then again, you wrote ` - Set CC=clang and CXX=clang in make.conf` in your bashrc hook.